set tarfile manually (proxmox 7 doesnt set it anymore, probably a bug)

tabnul opened the pull request:
I noticed the backups arent pushed to influx anymore after proxmox 7 update.
The size is not set and this is due to the $TARFILE variable not being set in backup-start, backup-end and log-end stages.
Also logging a bug to proxmox for this.

Error in log;
`{"error":"unable to parse 'proxmox,host=code,location=homeservers success=1,duration=95,speed=65,size=': missing field value"}
`
I set the tarfile manually by parsing it from the logs and it works.

tabnul committed:
set tarfile manually (proxmox 7 doesnt set it anymore, probably a bug)

moerphin commented:
Currently I don't have PVE7 test env, need a couple of days to create one.
Thank you, I'll check it.

tabnul committed:
Update vzdump2influx.sh

changed TARFILE to TARGET, TARFILE was deprecated and removed in proxmox V7 :

pve-manager (6.2-10) pve; urgency=medium

  * vzdump-hook-script: rename TARFILE to TARGET and add fallback to TARFILE
    for compatibility reasons

 -- Proxmox Support Team <support@proxmox.com>  Tue, 14 Jul 2020 10:28:42 +0200

tabnul commented:
i updated the file , see last commit description. and for reference;
https://forum.proxmox.com/threads/backup-hook-tarfile-variable-not-available-since-7-1-update.102270/#post-440769

it seems the issue was a bit more simple and the TARFILE variable was phased out. it works fine here!

tabnul committed:
influxdb2 http api compatible

tabnul committed:
Update vzdump2influx.sh

tabnul committed:
influxdb2 http api compatible

tabnul committed:
Update vzdump2influx.sh

tabnul committed:
Merge branch 'master' of github.com:tabnul/vzdump2influx

tabnul committed:
fix hostname

moerphin commented:
Thank you!
